features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -2077,7 +2077,7 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                     { applicationConfig.advancedConfigurations.showHybridFlowFeatureStatusChip && (
                                         <div className="oxygen-chip-div" >
                                             <Chip
-                                                label={ t("common:new").toUpperCase() }
+                                                label={ t("common:new") }
```

Comment:
If possible, shall we update this to get the label value from FeatureStatusLabel enum?

Reply:
ack

Reply:
fixed with https://github.com/wso2/identity-apps/pull/6617/commits/d1a5f52eb01ef2f57281cd03d75785ea99528e6b

Reply:
Thanks